tech/languages/php/php-installation.md

```diff
@@ -0,0 +1,59 @@
+---
+title: PHP
+page: php
+section: tech-languages
+---
+
+## PHP
+
+### PHP Installation
+
+To install PHP, simply type:
+
+```
+# dnf install php-cli
+```
+
+Above command will install latest stable PHP package, which includes the command-line interface executing PHP scripts, /usr/bin/php, and the CGI interface. 
+
+
+You might want to install some other packages, e.g. phpunit for unit tests or composer to manage dependencies of PHP projects. You can install them with:
+
+```
+# dnf install phpunit composer
+```
+
+### PHP modules
+PHP provides only a few set of extensions, other should be installed separately. For example, if you need "mysqli" extension, use:
+
+```
+# dnf install php-mysqli
+```
+
+### PHP Development Server
+
+PHP contains simple web server which can be used instead of full-featured web server. It is intended to be used only for development, not for a production environment. 
+
+You can start PHP Development Server by typing this in your project's root:
+
+```
+# php --server localhost:8080 --docroot  .
+```
+Server will be accessible on http://localhost:8080/ .
+
+
+### Frameworks
```

Comment:
I would make a separate page for each framework as they will help people to navigate on the site. Nobody will look on how to install Zend on the page with "PHP installation" title. Is it also common in the community to install these frameworks using composer? Perhaps both ways could be showed (as I did with installing Ruby on Rails).